### ISSUE TITLE
Command line args don't work as described

### ISSUE THREAD

**Issue by Vaguery** [erp12/pyshgp#54]
I tried `python examples/idea_of_numbers.py --population_size=200` and it errors out immediately with

~~~ text
Traceback (most recent call last):
  File "examples/idea_of_numbers.py", line 59, in <module>
    gp.evolution(error_func, problem_params)
  File "/usr/local/lib/python2.7/site-packages/pyshgp/gp/gp.py", line 148, in evolution
    params.grab_command_line_params(evolutionary_params)
  File "/usr/local/lib/python2.7/site-packages/pyshgp/gp/params.py", line 143, in grab_command_line_params
    while sys.argv[i+j].startswith('-'):
IndexError: list index out of range
~~~

(the `idea_of_numbers.py` file is my own)

**Comment by erp12** [comment]
Looks like this is another area of the docs that is out of date. The current pattern is actually:
```
python examples/idea_of_numbers.py -population_size 200
```
Hopefully this can be addressed alongside #46 in the near future.

**Comment by erp12** [comment]
Closing due to #46 and #65.
